src/user/mod.rs

```diff
@@ -378,6 +378,24 @@ pub fn updates(req: &mut Request) -> CargoResult<Response> {
     Ok(req.json(&R{ versions: versions, meta: Meta { more: more } }))
 }
 
+/// Handles the `GET /users/:user_id/stats` route.
+pub fn stats(req: &mut Request) -> CargoResult<Response> {
+    use  diesel::expression::dsl::sum;
+    let name = &req.params()["user_id"].parse::<i32>().ok().unwrap();
+    let conn = req.db_conn()?;
+    
+    let data = crate_downloads::table.filter(crate_downloads::crate_id.eq_any(
+        crate_owners::table.select(crate_owners::crate_id)
+            .filter(crate_owners::owner_id.eq(name))
+        )).select(sum(crate_downloads::downloads)).first::<i64>(&*conn)?; 
```

Comment:
Hey @sgrif thanks for taking a look at this. Your query makes total sense. I'll change it :)

> should this be limited to only users and not teams

Since we're sending `user_id` from the dashboard, we'll be basically querying `crate_owners` with `owner_kind` as `OwnerKind::User` only right?